packages/calling/src/CallingClient/calling/call.test.ts

```diff
@@ -346,7 +346,10 @@ describe('Call Tests', () => {
     expect(mockInternalMediaCoreModule.RoapMediaConnection).toBeCalledOnceWith(
       roapMediaConnectionConfig,
       roapMediaConnectionOptions,
-      expect.any(String)
+      expect.any(String),
+      expect.any(Function),
+      expect.any(Function),
+      expect.any(Function)
```

Comment:
These are added as we are using anonymous/arrow functions. Adding the function directly was not possible as we have a bit different logic to create the RtcMetrics object. Please see the screenshot attached below for reference.

<img width="905" alt="rtcmetrics-functions" src="https://github.com/user-attachments/assets/30d25627-d304-410d-8ac9-e679219663da">

Reply:
@rarajes2 I was able to avoid this error by doing this..
```
  private handleClose() {
    this.rtcMetrics.closeMetrics();
  }

  private handleSendMetric() {
    this.rtcMetrics.sendMetricsInQueue();
  }

  /* istanbul ignore next */
  /**
   * Initialize Media Connection.
   *
   * @param settings -.
   * @param settings.localAudioTrack - MediaStreamTrack.
   * @param settings.debugId - String.
   */
  private initMediaConnection(localAudioTrack: MediaStreamTrack, debugId?: string) {
    const mediaConnection = new RoapMediaConnection(
      {
        skipInactiveTransceivers: true,
        iceServers: [],
        iceCandidatesTimeout: ICE_CANDIDATES_TIMEOUT,
        sdpMunging: {
          convertPort9to0: true,
          addContentSlides: false,
          copyClineToSessionLevel: true,
        },
      },
      {
        localTracks: {audio: localAudioTrack},
        direction: {
          audio: 'sendrecv',
          video: 'inactive',
          screenShareVideo: 'inactive',
        },
      },
      debugId || `WebexCallSDK-${this.correlationId}`,
      (data) => this.rtcMetrics.addMetrics(data),
      this.handleClose,
      this.handleSendMetric,
    );
```

Is it not the same?

Reply:
I had tried this way as well. With add metrics also we should be able to simply pass `this.rtcMetrics.addMetrics` as this is equivalent to `(data) => this.rtcMetrics.addMetrics(data)` but was getting the issue with this keyword/object.